packages/styled-components/src/components/primitives/Tooltip/Tooltip.test.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import { Tooltip } from '.';
+
+describe('Tooltip', () => {
+  const testId = 'tooltip';
+  it('should render trigger', () => {
+    const trigger = 'trigger';
+    const tooltip = 'tooltip';
```

Comment:
move this one level up and remove from all the test cases

Reply:
Done